extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -185,24 +192,46 @@ public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Co
 
     @Override
     public void pause() {
-        try {
-            if (scheduler != null) {
-                scheduler.pauseAll();
+        if (!enabled) {
+            LOGGER.warn("Quartz Scheduler is disabled and cannot be paused");
+        } else {
+            try {
+                if (scheduler != null) {
+                    scheduler.standby();
+                }
+            } catch (SchedulerException e) {
+                LOGGER.warn("Unable to pause scheduler", e);
             }
-        } catch (SchedulerException e) {
-            LOGGER.warn("Unable to pause scheduler", e);
         }
     }
 
     @Override
     public void resume() {
-        try {
-            if (scheduler != null) {
-                scheduler.resumeAll();
+        if (!enabled) {
+            LOGGER.warn("Quartz Scheduler is disabled and cannot be resumed");
+        } else {
+            try {
+                if (scheduler != null) {
+                    scheduler.start();
+                }
+            } catch (SchedulerException e) {
+                LOGGER.warn("Unable to resume scheduler", e);
+            }
+        }
+    }
+
+    @Override
+    public boolean isRunning() {
+        if (!enabled || scheduler == null) {
```

Comment:
`|| scheduler == null` is IMHO redundant and adds no value.

Reply:
It can be `enabled = true` but scheduler can still be `null` (When no scheduled methods are found and `forceStart` config is false.)

Reply:
You're right. I'm sorry for the confusion...

Reply:
No problem at all, I just wanted to double check in case I oversee something.